### PR TITLE
VTCCA-6492 Updated the address line validation to be max 30 chars

### PR DESCRIPTION
**Pull request by joeyymoore**

### CODE REVIEW

File: app/form/Mappings.scala [hunk 1]
```diff
@@ -48,10 +48,12 @@ object Mappings extends DateMappings {
 
   val addressMapping: Mapping[Address] = mapping(
     "addressId" -> addressId,
-    "line1"     -> onlyIf(addressEnteredManually, text.verifying(nonEmpty, maxLength(36)))(""),
-    "line2"     -> text(maxLength = 36),
-    "line3"     -> text(maxLength = 36),
-    "line4"     -> text(maxLength = 36),
+    "line1" -> onlyIf(
+      addressEnteredManually,
+      text.verifying(nonEmpty, maxLength(30, "enrolment.address.maxLengthError")))(""),
```
Comment (WillBlakeney) [inline app/form/Mappings.scala:53]: I think line one always has a limit of 80 characters according to Mahmed's comment on [VTCCA-6394](https://jira.tools.tax.service.gov.uk/browse/VTCCA-6394)

**Comment by WillBlakeney** [comment]
When the user gets an error, the 'Enter address manually' link disappears. This means the user can only see the 'Postcode' field and not the error they actually received. I'm not sure if this is the scope creeping too much, but do you think we could make it so that the user can actually see their error?

**Comment by WillBlakeney** [comment]
Also, just want to check a ticket has been raised to fix the other fields on the page, add an error summary, fix changing the language etc etc?

**Comment by joeyymoore** [comment]
> Also, just want to check a ticket has been raised to fix the other fields on the page, add an error summary, fix changing the language etc etc?

Fix the other fields yes, fix the other stuff no. I can whip one up but theres so much wrong with it 😅

**Comment by joeyymoore** [comment]
> When the user gets an error, the 'Enter address manually' link disappears. This means the user can only see the 'Postcode' field and not the error they actually received. I'm not sure if this is the scope creeping too much, but do you think we could make it so that the user can actually see their error?

This was on my list of things that are wrong with the page, because of how the page works i dont think its a straight forward fix (like most other pages would be) so will need a separate ticket

File: app/form/Mappings.scala (hunk 1 above)
Comment (joeyymoore) [reply app/form/Mappings.scala:53]: okay i'll update the first line to be 80, and the corresponding error message too

**Comment by WillBlakeney** [comment]
Wow, it's annoying you can't make a thread with normal comments on a PR. But no worries 👍

**Comment by WillBlakeney** [comment]
rebuild

**Review by WillBlakeney — APPROVED** [review]

**Pull request merged**